koku/api/report/ocp/query_handler.py

```diff
@@ -291,6 +291,11 @@ def execute_query(self):  # noqa: C901
         }
         ordered_total.update(query_sum)
 
+        if self.is_csv_output:
```

Comment:
We already do this check up on line 274, is there a good reason not to include your if statement below there?